packages/generator/templates/page/index.tsx

```diff
@@ -30,7 +30,7 @@ export const __ModelNames__List = () => {
           {__modelNames__.map((__modelName__) => (
             <li key={__modelName__.id}>
               <Link
-                href={`/__parentModels__/${__parentModelId__}/__modelNames__/${__modelName__.id}`}
+                href={Routes.__ModelName__()}
```

Comment:
Should be this I think.
```suggestion
                href={Routes.Show__ModelName__Page()}
```